### DISCUSSION TITLE
MOD: Add An Oldham Coupler - Suggestion

### DISCUSSION THREAD

**Discussion by aheil** [1coderookie/Kobra2NeoInsights#6] [General]
Thanks a lot for this great page, it helped me a lot of working on my Neo 2. 

Due to worn off parts I added an Oldham coupler. After that my print head felt down just by gravity all the time, once the motor is turned off. I came acrosse the following video explaining the mechanism of the anti backslash nut and why it might make sense to but it on upwards. As this section is mission, you might want to have a look. 

At the very end, if installed on the top it should prevent the print head slipping down by gravity.  

[https://youtu.be/kmqLz-AmzVw?si=27_cX8CJPzY4Dtgx](https://youtu.be/kmqLz-AmzVw?si=27_cX8CJPzY4Dtgx)

**Comment by 1coderookie** [comment]
Thanks for reaching out! Glad it's helpful for you :)  
I'm aware of that topic/general discussion and I actually already started a textblock about that when I set up the section about the Oldham Coupler ("Upwards or Downwards?"), but it seems that I forgot to complete that one.. :(   
The only time I had an issue with a dropping gantry was when I installed those other spring-loaded anti-backlash nut incorrectly, since I left a little gap between those two parts. But apart from that, I never had an issue with the nut and O.C. facing downwards, so I actually didn't bother comparing. But if there's enough clearance at the top and the end of the nut won't come off the end of the lead screw, then it's definitely worth a try.
